common/src/stack/command/stack/commands/set/host/power/imp_ssh.py

```diff
@@ -0,0 +1,41 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+import stack.commands
+import shlex
+from stack.util import _exec
+from stack.exception import CommandError
+
+class Implementation(stack.commands.Implementation):
+	def run(self, args):
+		host = args[0]
+		cmd = args[1]
+
+		if cmd == 'on':
```

Comment:
This looks like it should be an `if: ... elif: ...` chain since `cmd` should only ever be one of these values. The other option is a `return` statement in each condition.